sky/packages/sky/lib/src/fn3/editable_text.dart

```diff
@@ -195,13 +195,16 @@ class EditableTextState extends State<EditableText> {
     else if (!config.focused && _cursorTimer != null)
       _stopCursorTimer();
 
-    return new _EditableTextWidget(
-      value: config.value,
-      style: config.style,
-      cursorColor: config.cursorColor,
-      showCursor: _showCursor,
-      onContentSizeChanged: config.onContentSizeChanged,
-      scrollOffset: config.scrollOffset
+    return new SizedBox(
```

Comment:
I feel ConstrainedBox is probably the better API to use for this, but BoxConstraints doesn't have a way to express an unconstrained height with a tight-infinite width currently without explicitly specifying "double.INFINITY" which is something I've vaguely tried to do outside of BoxConstraints. Maybe we need a BoxConstraints.loosenHeight() so we could do `new BoxConstraints.expand().loosenHeight()`...